source/opt/basic_block.h

```diff
@@ -45,7 +45,7 @@ class BasicBlock {
   // Appends an instruction to this basic block.
   inline void AddInstruction(std::unique_ptr<Instruction> i);
   // The label starting this basic block.
-  Instruction& Label() { return *label_; }
+  Instruction* GetLabelInst() { return &*label_; }
```

Comment:
This was part of an aborted attempt to eliminate unused control flow using instruction pointers. I will eliminate for now.